operator/v1/types_network.go

```diff
@@ -339,6 +345,40 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type ExportNetworkFlows struct {
```

Comment:
I think you can set `+kubebuilder:validation:MinProperties=1` somewhere (either above this type or from where it's included in `NetworkSpec`) to indicate that at least one of these optional fields needs to be set in order for it to be considered valid. ie, you can't do:

    spec:
      exportNetworkFlows: {}